src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -131,6 +133,9 @@ function parseEndpoints(
       });
       triggered = { httpsTrigger: {} };
       copyIfPresent(triggered.httpsTrigger, ep.httpsTrigger, "invoker");
+    } else if (backend.isCallableTriggered(ep)) {
+      triggered = { callableTrigger: {} };
+      ep.labels = { ...ep.labels, "deployment-callabled": "true" };
```

Comment:
Rather than setting the labels here, let's set them in the gcp/cloudfunctions and gcp/cloudfunctionsv2 `endpointToFunction` methods.